tests/expr_and_series/dt/datetime_attributes_test.py

```diff
@@ -120,11 +126,10 @@ def test_to_date(request: pytest.FixtureRequest, constructor: Constructor) -> No
             "pandas_nullable_constructor",
             "cudf",
             "modin_constructor",
-            "pyspark",
         )
     ):
         request.applymarker(pytest.mark.xfail)
-    dates = {"a": [datetime(2001, 1, 1), None, datetime(2001, 1, 3)]}
+    dates = {"a": [datetime(2001, 1, 1), datetime(2001, 1, 3)]}
```

Comment:
Removal of None is due to the fact that otherwise pyspark ends up creating a column of struct type. We might need to address this in the constructor?

Reply:
🤔 yeah might be good to see if we can address the constructor so it doesn't go via pandas, that might in general be good going forwards